unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/RPC/Services/SceneControllerService/Tests/SceneControllerServiceShould.cs

```diff
@@ -241,6 +246,126 @@ await rpcClient.SendCrdt(new CRDTSceneMessage()
             });
         }
 
+        [UnityTest]
+        public IEnumerator CallGetCurrentStateWithoutStoredState()
```

Comment:
You can use `async Task` directly without coroutines and just with `[Test]` attribute, please try and check if it works in this case.

Reply:
great to see async tests now working properly, thanks!
changed https://github.com/decentraland/unity-renderer/pull/3975/commits/dc0a7120a315d281f79876592657756f149dc5c0